<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1078</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string extracomment="Melhor Playlister do mundo!">DoReMi - Seu SetList de Spotify</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="mopW_PushButton_TestaSpotify">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Teste</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="mopW_TextBrowser_Out">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>751</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="mopW_PushButton_Crypto">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>10</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Testa Crypto</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mopW_PushButton_ConfigClienteSpotify">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Configurar Cliente Spotify</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="mopW_LineEdit_Search">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>311</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Procure no Spotify!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mopW_PushButton_Search">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>40</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>CadÃª</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>70</y>
      <width>371</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>40</y>
      <width>331</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_3">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>40</y>
      <width>311</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1078</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
